analytics/urls.py

```diff
@@ -26,6 +26,8 @@
     # User-visible stats page
     url(r'^stats$', analytics.views.stats,
         name='analytics.views.stats'),
+    url(r'^info$', analytics.views.get_info,
```

Comment:
Should have clarified; this should be a server admin (user_profile.is_staff) visible page only. 

Tim, thoughts on URL? Maybe /activity/support?

Reply:
Yeah, I think /activity/support seems ifne.